packages/utils/src/validation/validatorFunctions.js

```diff
@@ -286,13 +288,13 @@ export const constructIsNotPresentOr = validatorFunction => (value, object, key)
 };
 
 export const constructIsLongerThan = minLength => value => {
-  if (value.length < minLength) {
+  if (value.length <= minLength) {
```

Comment:
Saw your password length changes @jaskfla and decided to push this up since we incorrectly allow creating users with only 8 char passwords (I know this since I create test users all the time with them :sweat_smile:)

Reply:
Ooh, good catch—thanks